### DISCUSSION TITLE
CMT portal initial submission

### DISCUSSION THREAD

**Discussion by dskhanirfan** [FETS-AI/Challenge#100] [General]
Hi 
Is there any format or any blueprint to submit the initial short paper submission on the CMT system.
We only have to mention which task we are working on right?

**Comment by sarthakpati** [comment]
> Is there any format or any blueprint to submit the initial short paper submission on the CMT system.

Nope.

> We only have to mention which task we are working on right?

Yes, and any other questions that the CMT system would present.

**Reply by mzenk** [reply]
Just to clarify, the *initial* submission does not need to have any specific format, but the final manuscript should follow the [LNCS template](https://www.springer.com/us/computer-science/lncs/conference-proceedings-guidelines) as it says on the [website](https://fets-ai.github.io/Challenge/participate/). Or is this information outdated @sarthakpati ?

**Reply by dskhanirfan** [reply]
if you guys can create a template for the statement which highlights which task we want to submit. That would be great.

**Reply by sarthakpati** [reply]
Tagging @ujjwalbaid0408

**Comment by ujjwalbaid0408** [comment]
Hi @dskhanirfan,

If you try to initiate a submission on CMT, there are a couple of questions that you need to answer and one of the questions is about the task. Please make sure to submit the short paper in LNCS format by July, 26.
